balancer/xds/edsbalancer/balancergroup.go

```diff
@@ -339,6 +345,13 @@ func (lrp *loadReportPicker) Pick(ctx context.Context, opts balancer.PickOptions
 		td := done
 		done = func(info balancer.DoneInfo) {
 			lrp.loadStore.CallFinished(lrp.id, info.Err)
+			if load, ok := info.ServerLoad.(*orcapb.OrcaLoadReport); ok {
```

Comment:
The OrcaLoadReport is not applied atomically, will this be any issue for the traffic director?

Reply:
I think this should be OK.
This is the best we can do to avoid adding contention for all RPCs.